### ISSUE TITLE
SSL Connection

### ISSUE THREAD

**Issue by benitovc** [paritytrading/philadelphia#73]
Is it possible to change the connection to use SSL Socket?

**Comment by jvirtanen** [comment]
No, not right now. The primary use case for Philadelphia is in private networks in colocated data centers. You can deploy e.g. [stunnel](https://www.stunnel.org/) in front of your application.

**Comment by jvirtanen** [comment]
Philadelphia 2.0.0 will have the capability to interface with e.g. [TLS Channel](https://github.com/marianobarrios/tls-channel).
